content/en/docs/getting-started/_index.html

```diff
@@ -0,0 +1,266 @@
+{{<highlight javascript>}}
+  module "eks-jx" {
+    source  = "jenkins-x/eks-jx/aws"
+    vault_user=""
+  }
+{{</highlight>}}
+This setup uses defaults parameter values. You can modify them, more <a href="docs/getting-started/setup/create-cluster/eks/#inputs">details here</a>.<br>
+<div class="card text-white bg-primary mb-3" style="max-width: 70rem;">
+  <div class="card-header">IAM User</div>
+  <div class="card-body">
+    <h4 class="card-title">Specify existing IAM User</h4>
+    <p class="card-text">You will need to provide a existing IAM user name for Vault. The specified user's access keys are used to authenticate the Vault pod against AWS. The IAM user does not need any permissions attached to it. This Terraform Module creates a new IAM Policy and attaches it to the specified user. <br>
+      For more information refer to <strong><a class="card-link text-white" href="/docs/getting-started/setup/boot/clouds/amazon/#configuring-vault-for-eks">Configuring Vault for EKS</a></strong>.</p>
+  </div>
+</div>
+
+
+          <p>2 . Provision the cluster using Terraform. The output of this command will include a file <strong>jx-requirements.yml</strong> which<br/>
+            you will use in <strong>Step 3: Install Jenkins X on your cluster</strong> </p>
+{{<highlight bash>}}
+$ terraform init && terraform apply
+{{</highlight>}}
+        </div>
+        <div class="tab-pane fade" id="azure">
+          <div class="col-sm-4">
+              <div class="card">
+                <div class="card-body">
+                  <a href="/docs/getting-started/setup/create-cluster/azure/"><img src="https://www.pulumi.com/logos/tech/azure.svg"/></i></a>
+                </div>
+              </div>
+          </div>
+            <h4>Use Azure (AKS) Terraform Module</h4>
+            <p>instructions on how to here.</p>
+        </div>
+          <div class="tab-pane fade" id="openshift">
+          <p>Coming soon!</p>
+        </div>
+      </div>
+      </div>
+    </div>
+  <p>&nbsp;</p>
+  <div style="row">
+  <div class="col col-lg-12">
+    <p>&nbsp;</p>
+    <h2>Step 3: Install Jenkins X on your cluster</h2>
+    
+    <p>You have provisioned your Kubernetes cluster, you are ready to install Jenkins X on top of the cluster.  To do that, you will use the <strong>JX Boot</strong> CLI command.</p>
+    <div class="alert alert-dismissible alert-primary"> 
+      <strong>What is JX Boot?</strong> With JX Boot you Install, configure or upgrade Jenkins X via GitOps and a Jenkins X Pipeline.  This pipeline resides within your
+      config git repository, also known as a <a class="alert-link" href="/about/concepts/components/#permanent-environments">permanent environment</a>
+    </div>
+  {{<highlight bash>}}
+    # the parameter here, is the requirements yaml file
+    # which was saved in the same directory you executed terraform apply.
+    $ jx boot -r jx-requirements.yml
```

Comment:
no '$' to indicate a shell prompt. We should try to be consistent. We don't have it in the rest of the documentation.

Reply:
You should not execute `jx boot` from the same directory as you execute `terraform` from. You should form **another** empty directory execute:

`jx boot -r <path-to-jx-requirements.yml`

Eventually, you want to create a git repo for your Terraform config and it should / needs to be separate from your boot repo.

Reply:
That's still an issue here.